<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile</class>
 <widget class="QWidget" name="profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Profile</string>
     </property>
     <widget class="QLabel" name="label_Username">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_Age">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_Gender">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_Phone">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_wins">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_loses">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_draws">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Wins</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Loses</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Draws</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Rank</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Rank">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Profile</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
